### ISSUE TITLE
About the model training

### ISSUE THREAD

**Issue by CXY-Likescoding** [intrepidChw/ASAP-Net#2]
Thanks for opening source the project. I've downloaded your code and trained the model './Synthia_experiments/semantic_seg_synthia/models/model_PNv2_ASAP-1.py' following the 'readme.md', but the loss function didn't convergence after dozens of epochs' training. Am I missing something?

**Comment by hwcao17** [comment]
It worked well for me. Did you follow the instructions in README?
